site/docs/0.0/content/font.md

```diff
@@ -0,0 +1,749 @@
+- Worry-free font hosting.
+- No managing department or team-level creative cloud accounts to ensure your
+font isn't tied to one team-member
+- Leverages browser caching across other domains and subdomains that use the
+shared assets.
+- You can easily switch to a self-managed model and back again.
+- Centralized documentation on how to integrate into other shared assets.
+
+**Disadvantages**
+- Less flexibility in regards to which fonts are available to your project.
+- Workflow challenges with making updates to centrally managed web fonts.
+
+<!--Using shared standard assets is inherently faster across multiple-->
+<!--domains, any deviation to using shared standard assets should be considered-->
+<!--carefully by researching traffic flow between institutional web properties, and-->
+<!--not just initial page load times on one site.  In other words all sites that-->
+<!--link to arizona.edu domain, typically share users with the parent domain, and-->
+<!--other subdomains. Browsers cache assets based on the address it is downloaded-->
+<!--from, so if all subdomains share the same asset, the end user's browser will-->
+<!--access a cached copy without downloading it again across those subdomains.-->
+
+
+## Global settings
+For projects not using Arizona Bootstrap, [follow these instructions](#instructions) on how to add Proxima Nova to your website. Be aware that Arizona Bootstrap reference links come "pre-loaded" with the Proxima Nova font suite. In other words, if you're using Arizona Bootstrap, font installation instructions are not required.
+
+
+### **Native Fallback Fonts**
+The fallback web fonts (Verdana, Geneva) have been dropped in Arizona Bootstrap
+4 and replaced with a “native font stack” for optimum text rendering on every
+device and OS. Read more about
+[native font stacks in this Smashing Magazine article](https://www.smashingmagazine.com/2015/11/using-system-ui-fonts-practical-guide/).
+
+```
+$font-family-sans-serif:
+  proxima-nova,
+  // Safari for macOS and iOS (San Francisco)
+  -apple-system,
+  // Chrome < 56 for macOS (San Francisco)
+  BlinkMacSystemFont,
+  // Windows
+  "Segoe UI",
+  // Android
+  "Roboto",
+  // Basic web fallback
+  "Helvetica Neue", Arial, sans-serif,
+  // Emoji fonts
+  "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol" !default;
+```
+  This `font-family` is applied to the `<body>` and automatically inherited
+  globally throughout Bootstrap. To switch the global `font-family`, update
+  `$font-family-base` and recompile Arizona Bootstrap.
+
+## How to Use
+  The Proxima Nova font suite is available for official use by University of Arizona
+  employees through a license with Adobe Typekit.
+
+### Reference link
+  Arizona Bootstrap uses a Creative Cloud Library with shared governance to
+  manage our font reference links within Arizona Bootstrap. Alternatively, place
+  this line of code at the top of your main stylesheet.
+
+  ```@import "https://use.typekit.net/tgs5qmz.css";```
```

Comment:
@trackleft can you please add...

- [ ]  WARNING this shouldn't be used

Reply:
we don't yet support wide usage of the font from a centralized reference link

Reply:
Added a BEWARE! alert.